src/app/rundown/components/rundown-header/rundown-header.component.html

```diff
@@ -0,0 +1,23 @@
+<div [cdkContextMenuTriggerFor]="contextMenu" class="c-rundown-header" [class.c-rundown-header--active]="rundown.isActive">
+    <ng-template #contextMenu>
+        <sofie-rundown-header-context-menu class="c-context-access-surface" [rundown]="rundown"> </sofie-rundown-header-context-menu>
+    </ng-template>
+
+    <sofie-logo></sofie-logo>
+    <sofie-rundown-header-panel [label]="rundownPath">{{rundownName}}</sofie-rundown-header-panel>
```

Comment:
The label for this one is using camelCase, while the other labels use normal text.
Is there any reason for this?

Reply:
Is it because it uses the attribute in the component class?

Reply:
Looking at it further i am sure it is the reason.

But that now means that this is a magic string, unless i am mistaken.
Is this how it has been done in other places too?
I am not a fan of magic strings...

Reply:
`rundownPath` is a variable of `RundownHeaderComponent` that is passed to the input parameter `label` of `RundownHeaderPanelComponent`.